javascripts/govuk/analytics/google-analytics-universal-tracker.js

```diff
@@ -20,6 +22,10 @@
       sendToGa('set', 'displayFeaturesTask', null)
     }
 
+    function stripLocationPII () {
+      sendToGa('set', 'location', stripPIIFromString(window.location.href.toString()))
```

Comment:
I think `window.location.href` is already a string so we don't need to call `toString`.

Reply:
Ah, of course.  I was thinking of `window.location`, which is an Object.